doc/users/next_whats_new/2019-03-25-mplot3d-projection.rst

```diff
@@ -4,6 +4,20 @@
 Plots made with :class:`~mpl_toolkits.mplot3d.axes3d.Axes3D` were previously
 stretched to fit a square bounding box. As this stretching was done after
 the projection from 3D to 2D, it resulted in distorted images if non-square
-bounding boxes were used.
+bounding boxes were used.  As of this release, this no longer occurs by default.
 
-As of this release, this no longer occurs.
+We have extended the functionality of
+`~mpl_toolkits.mplot3d.axes3d.Axes3D.set_aspect` to accept the modes
+``'auto_Pb'``, which uses a fixed ratios between the sizes, in display
```

Comment:
```suggestion
``'auto_pb'``, which uses a fixed ratios between the sizes, in display
```
?

Also, what does „pb“ stand for? I have no association for that and the docs don‘t give. Hint either.

Reply:
"pseudo-box", I'm going to look into if we have a mode in the 2D case which matches this (fixed aspect ratio of the axes box but no adjustment of the data limits).

Reply:
Unfortunately we have other instances of `pb` for `parent_bbox`.  Not sure if those are public facing though.   

Fixing the box aspect is basically #14917?

Reply:
Ah, I now feel less bad about not knowing `box_aspect` existed as it is a new feature :)

On the bright side (some what ironically) if you use `set_box_aspect` with normal axes it is also broken:

```python
import matplotlib.pyplot as plt
fig, ax = plt.subplots()
ax.set_box_aspect(2)
fig.savefig('/tmp/womp_womp.png', bbox_inches='tight', facecolor='teal')
```
![womp_womp](https://user-images.githubusercontent.com/199813/83163305-e3982300-a0d8-11ea-9971-86bad7abdb88.png)

This means we can drop `auto_pb` and find a uniform way to suppress the rescaling (I suspect via the method monkey patching context manager or adding a "don't touch" aspect mode.

Reply:
I'm also uncertain what pseudo-box means, as it sounds somewhat internal.

Reply:
:+1: Thank you for pushing back on this @timhoffm !  My original solution was a bit of a hack, what I just pushed is much better :)

Reply:
and @QuLogic who's comment I didn't see until after I posted (always remember to refresh the browser...)